CONTRIBUTING.md

```diff
@@ -18,12 +18,25 @@ Our Rust projects use the standard Rust coding style, and should all have tests.
 First, make sure you have an up-to-date stable Rust, with the necessary tools. You can install Rust using `rustup`:
 
 ```sh
+
+# If you have linux system rust installed, remove it first
+sudo apt-get remove rustc
+
 # On Linux and MacOS. Others see https://rustup.rs/
 curl https://sh.rustup.rs -sSf | sh
 
 # Install Rust tools for code formatting and lints.
 rustup component add rustfmt
 rustup component add clippy
+
+# To switch to rust nightly
+rustup toolchain install nightly
+rustup default nightly
```

Comment:
You shouldn't have to do this. Instead, just run `cargo $COMMAND` in the `dbcrossbar` directory, and `cargo` will make sure the Rust compiler specified in `rust-toolchain` is actually installed.